### PR TITLE
Add FS reconstruction class

### PR DESCRIPTION
**Pull request by MadsJensen**
This is the initial FreeSurfer addition to the stormdb process module. 

Now it only works for all subjects in a study

### CODE REVIEW

**Comment by cjayb** [comment]
Thx, kigger paa asap...
Can see issue with `submit_to_cluster`. Might try to fix in that branch, then rebase this. 
Need to think about API for all classes in `process`, don't want different behaviour...

**Comment by MadsJensen** [comment]
Single subject method has been merge into this branch

File: stormdb/process.py [hunk 1]
```diff
@@ -491,9 +499,46 @@ def all_subjects(self):
...
                     self.info["cmd"] += [cmd]
                 else:
-                    print ("\nProblen with T1 for subject: %s" % subject,
-                           "Either none or multiple T1\'s "
-                                     "present for subject")
+                    print("\nProblen with T1 for subject: %s" % subject,
+                          "Either none or multiple T1\'s "
+                          "present for subject")
+
+    def single_subject(self, subject=None, description="t1*):
```
Comment (cjayb) [inline stormdb/process.py:506]: This is broken (missing " after *)

**Comment by cjayb** [comment]
Now we run

```
from stormdb.access import Query
from stormdb.process import Maxfilter
for sub in qy.get_subjects():
   <<<some logic to define io, incl. more calls to qy.*-methods>>>
   mf.build_maxfilter_cmd(in_fname, out_fname)

mf.submit_to_isis(n_jobs=4)  # now len(cmd) can be >> 1
```

so unless there are good arguments, I'd like to keep the API the same. For Freesufer, this would mean something like:

```
from stormdb.process import Freesurfer  # or FS_reconstruction
fs = Freesurfer('MINDLAB20XX_MEG-YourProject')
for sub in qy.get_subjects():
   series = db.filter_series(description='t1*', subj_ids=sub, modalities="MR")
   fs.build_freesurfer_cmd(dicom=series[0]['files'][0], fs_dir=some_dir)

fs.submit_to_isis(n_jobs=4)  # now len(cmd) is len(qy.get_subjects())
```

I prefer not to have the various "submittable" objects depend on `Query`, _i.e._, that they just do one thing: build a list of command-lines and submit them to an (appropriate) queue.

Perhaps we should then have a `stormdb.wrappers`, or smth, which could be called as: `from stormdb.wrappers.FS import process_all_subjects` and that would then make the loops you have in the present code.

Talk more in person/on whiteboard...

**Comment by cjayb** [comment]
on second (third) thought, see meeg-cfin/configurations#9

**Comment by cjayb** [comment]
Closed by #32

**Pull request closed without merging**